qcodes/instrument_drivers/yokogawa/GS200.py

```diff
@@ -352,6 +357,21 @@ def _ramp_source(self, ramp_to: float, step: float, delay: float) -> None:
         self.output_level.step = saved_step
         self.output_level.inter_delay = saved_inter_delay
 
+    def _exclude_snapshot(self, mode: str) -> bool:
```

Comment:
it seems to me that this method is not really needed if it's only used at init of the instrument. would you agree?

Reply:
@astafan8 I have removed the _exclude_snapshot method and excluded current and current_range from the snapshot at init.